.orchestra/config/components/vscode-web.yml

```diff
@@ -57,15 +57,12 @@ builds:
       fi
       popd &> /dev/null
 
-      export HARD_FLAGS_COMPILE="-I${ORCHESTRA_ROOT}/include/node $HARD_FLAGS_COMPILE"
-
       # Install dependencies, this process is mostly offline with the following exceptions
       # (to the best of our knowledge):
-      # * node-gyp downloading header tarballs to compile native modules for node and electron
-      #   (This can be fixed once node is within orchestra)
```

Comment:
This means we no longer need `node-gyp` from the host operating system?

Reply:
Yes, it's bundled with `nodejs` as well as `npm`